Willning/DLOplanTool
# DLO-50 Personal Look and Feel

### **Related Issue/Keyword:**
[DLO-50](https://trello.com/c/Duw8ZU1d)
### **Description:**
give the planner a more personalised feel. This could be through custom avatar, custom colours, etc.
### **Testing:**
`npm install` 
 - check that nothing breaks,
 - check that colours all change correctly
 - check readability etc
> I know that when changing the colours the "D" on the logo in the header bar does not change until the user clicks OKAY. Not sure why this is, but not major 
### **Checklist:**
